services/property-svc/internal/property-value/commands/v1/attach_property_value.go

```diff
@@ -0,0 +1,45 @@
+package v1
+
+import (
+	"context"
+	"github.com/google/uuid"
+	"hwdb"
+	"hwes"
+	"property-svc/internal/property-value/aggregate"
+	"property-svc/repos/property_value_repo"
+)
+
+type AttachPropertyValueCommandHandler func(ctx context.Context, propertyValueID uuid.UUID, propertyID uuid.UUID, value interface{}, subjectID uuid.UUID) error
+
+func NewAttachPropertyValueCommandHandler(as hwes.AggregateStore) AttachPropertyValueCommandHandler {
+	return func(ctx context.Context, propertyValueID uuid.UUID, propertyID uuid.UUID, value interface{}, subjectID uuid.UUID) error {
+		propertyValueRepo := property_value_repo.New(hwdb.GetDB())
+		var a *aggregate.PropertyValueAggregate
+
+		// TODO: ensure eventual consistency by not using the projection here
```

Comment:
Eventual consistency can not be "ensured", we already have it

Reply:
right